providers/base/metrics/provider.go

```diff
@@ -19,6 +19,8 @@ const (
 	ProviderTypeLabel = "type"
 	// StatusLabel is a label for the status of a provider response.
 	StatusLabel = "status"
+	// ErrorLabel is a label for and error of a failed provider response.
+	ErrorLabel = "error"
```

Comment:
I would generally recommend not logging direct errors from the providers since these may vary significantly across the board. We should set up our instrumentation to notify us anytime the zapper logger does get an error in the response tho. I would talk to @Zygimantass about this.

Reply:
I agree w this.  Will push changes where we can standardize with error codes